source/voice_control/assist_create_open_ai_personality.markdown

```diff
@@ -16,63 +16,65 @@ related:
     
 ---
 
-You can give your voice assistant personality by using an AI conversation agent. Currently, this works with the **OpenAI Conversation** or the **Google Generative AI** integration.
+You can give your voice assistant personality by using an AI conversation agent.
 
-For this tutorial, we will work with OpenAI. This requires an OpenAI account. For what we do here, the free trial option is sufficient.
+## What can I do in Assist with AI exactly?
 
-## Demos
+- Pick the LLM provider of your choice, either local or cloud, as long as it has a conversational agent.
+- Select a personality based on a prompt.
+- Get replies with the character's personality you defined.
+- Perform Home Assistant intents (turn on-off lights, etc), as long as Assist is correctly configured as per our [best practices](/voice_control/best_practices).
```

Comment:
```suggestion
- Perform Home Assistant intents (such as turning on-off lights), as long as Assist is correctly configured as per our [best practices](/voice_control/best_practices).
```
we avoid using [etc](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/etc)